src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier extends ColorSupplier {
```

Comment:
FigureSupplier is not a ColorSupplier
```suggestion
public class FigureSupplier {
```